app/Http/Controllers/AdminController.php

```diff
@@ -237,7 +237,7 @@ public function eventsCsv()
             }
 
             $participants += $party->pax;
-            $hours_volunteered += (($party->volunteers > 0 ? $party->volunteers * 3 : 12 ) + 9);
+            $hours_volunteered += $party->hoursVolunteer();
```

Comment:
Typo?  Looks like it's hoursVolunteered()

Reply:
Yes.